Base.rte/Actors/Brains/Case/BrainCase.ini

```diff
@@ -94,6 +94,7 @@ AddActor = Actor
 			FilePath = Base.rte/Actors/Brains/Case/Sounds/BrainPop.flac
 	Team = 0
 	Health = 100
+	Robotic = 1
```

Comment:
Should probably be changed to `Synthetic`, but doesn't have to be done now (to maximize mod breakage in the future:tm: lelelelel)

Also, wouldn't the brain be both?

Reply:
Hmmm, I dunno if brain counts as organic? I guess I kinda figured it's hermetically sealed so it doesn't really count, but techincally there is organic shit in there so maybe.

As for renaming this, I guess I can. You should post it somewhere so people can bitch at you about it though :D

Reply:
Meh, can be both, can be either, can be neither. \_/¯( ͜。 ͡ʖ ͜。)¯\\\_
And you can always blame me for whatever. lel